lib/state/revision/reducer.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REVISION_IS_VIEWING,
+	REVISION_SELECT,
+} from '../action-types';
+
+export const isViewingRevisions = ( state = false, { type, isViewing } ) =>
+	REVISION_IS_VIEWING === type ? isViewing : state;
+
+export const selectedRevision = ( state = null, { type, revision } ) =>
+	REVISION_SELECT === type ? revision : state;
+
+export default combineReducers( {
+	isViewingRevisions,
```

Comment:
could not the boolean value of whether or not we are viewing a revision be captured in the presence or absence of state in the `selectedRevision`?

I'm not sure we need two values here when we could probably pull this out from a selector.